client/bio-webclient/src/main/kotlin/ac/uk/ebi/biostd/client/integration/web/SubmitClient.kt

```diff
@@ -36,7 +37,8 @@ interface SubmitClient :
     GeneralOperations,
     DraftSubmissionOperations,
     ExtSubmissionOperations,
-    PermissionOperations
+    PermissionOperations,
+    StatsOperations
```

Comment:
Trailing comma

Reply:
not allow in interface implementations :(